ruby/frontend/frontend.ru

```diff
@@ -44,10 +38,10 @@ end
 
 class GreetingsController < ActionController::Base
   def index
-    name = NameClient.get_name
-    message = MessageClient.get_message
+    @name = NameClient.get_name
+    @message = MessageClient.get_message
     Tracer.in_span("🎨 render message ✨") do |span|
-      render plain: "Hello #{name}, #{message}"
+      render inline: "Hello <%= @name %>, <%= @message %>"
```

Comment:
Interesting. Does this change the instrumentation values somehow, with the intervening ERB stuff? Or is it just being more Rails-ier?

Reply:
This turns it into an inline ActionView template and so will trigger any ActionView instrumentation. This was the reason for my bundling shenanigans with [an alternate version of OTel Ruby.](https://github.com/open-telemetry/opentelemetry-ruby/pull/796)

Reply:
![Neat](https://morbotron.com/video/S05E05/eoG7IIil7xrSvft7G9bg3HBTGo8=.gif)